support-workers/src/test/scala/com/gu/support/workers/LambdaSpec.scala

```diff
@@ -7,9 +7,7 @@ import org.scalatest.{AsyncFlatSpec, FlatSpec, Matchers}
 
 abstract class LambdaSpec extends FlatSpec with Matchers with MockContext
 
-abstract class AsyncLambdaSpec extends AsyncFlatSpec with Matchers {
-  implicit override def executionContext = scala.concurrent.ExecutionContext.Implicits.global
```

Comment:
this is the crucial change - now we use the serial execution context built into the scala async test framework.  This doesn't seem to hit the bug where it's running more than one test at the same time for some reason, possibly due to us Awaiting during the tests and it starting the next test before unblocking the Await and completing the previous one.

Reply:
Looks good, I assume there was a reason why this is overridden here but I can't remember why it was and if everything still functions, then 👍